howto/setup-pub-sub-message-broker/setup-kafka.md

```diff
@@ -1,3 +1,53 @@
 # Setup Kafka
 
-Content for this file to be added
+## Installing kafka with existing make target for development
```

Comment:
Could you please structure it like other docs under setup-pub-sub-message-broker with sections like Locally, Kubernetes. It would be good if it doesn't point to e2e-test
eg. https://github.com/dapr/docs/blob/master/howto/setup-pub-sub-message-broker/setup-nats.md

Reply:
I refactored